conversion_rules_export/markdown/aws-cur.md

```diff
@@ -89,9 +89,9 @@
 | Provider                   |                1 | NA                           | Not Defined          | ASSIGN_STATIC_VALUE | static_value: AWS                                                                                                                              |
 | Region                     |                1 | product/region               | Not Defined          | RENAME_COLUMN       |                                                                                                                                                |
 | ResourceId                 |                1 | lineItem/ResourceId          | string               | RENAME_COLUMN       |                                                                                                                                                |
-| ServiceCategory            |                1 | lineItem/ProductCode         | Not Defined          | LOOKUP              | destination_value: aws_category                                                                                                                |
+| ServiceCategory            |                1 | lineItem/ProductCode         | Not Defined          | LOOKUP              | destination_value: ServiceCategory                                                                                                                |
 |                            |                  |                              |                      |                     | reference_dataset_path: conversion_configs/aws/mapping_files/aws_category_mapping.csv                                                          |
-|                            |                  |                              |                      |                     | source_value: product_code                                                                                                                     |
+|                            |                  |                              |                      |                     | source_value: line_item_product_code                                                                                                                     |
```

Comment:
Same change as above, the source column is lineItem/ProductCode for aws-cur

Reply:
Made the change.